nautobot/core/settings.py

```diff
@@ -532,3 +532,32 @@
 CELERY_RESULT_SERIALIZER = "nautobot_json"
 
 CELERY_BEAT_SCHEDULER = "nautobot.core.celery.schedulers:NautobotDatabaseScheduler"
+
+#
+# Custom branding (logo and title)
+#
+
+# Branding logo locations. The logo takes the place of the Nautobot logo in the top right of the nav bar. The filepath should be relative to the `MEDIA_ROOT`.
+BRANDING_FILEPATHS = {
+    "logo": os.getenv("NAUTOBOT_BRANDING_FILEPATHS_LOGO", None),  # Navbar logo
+    "favicon": os.getenv("NAUTOBOT_BRANDING_FILEPATHS_FAVICON", None),  # Browser favicon
+    "icon_16": os.getenv("NAUTOBOT_BRANDING_FILEPATHS_ICON_16", None),  # 16x16px icon
+    "icon_32": os.getenv("NAUTOBOT_BRANDING_FILEPATHS_ICON_32", None),  # 32x32px icon
+    "icon_180": os.getenv(
+        "NAUTOBOT_BRANDING_FILEPATHS_ICON_180", None
+    ),  # 180x180px icon - used for the apple-touch-icon header
+    "icon_192": os.getenv("NAUTOBOT_BRANDING_FILEPATHS_ICON_192", None),  # 192x192pc icon
+    "icon_mask": os.getenv(
+        "NAUTOBOT_BRANDING_FILEPATHS_ICON_MASK", None
+    ),  # mono-chrome icon used for the mask-icon header
+}
+
+# Title to use in place of "Nautobot"
+BRANDING_TITLE = os.getenv("NAUTOBOT_BRANDING_FILEPATHS_ICON_MASK", "Nautobot")
+
+# Branging URLs (links in the bottom right of the footer)
+BRANDING_URLS = {
+    "code": os.getenv("BRANDING_URLS_CODE", "https://github.com/nautobot/nautobot"),
+    "docs": os.getenv("BRANDING_URLS_DOCS", "https://nautobot.readthedocs.io/"),
+    "help": os.getenv("BRANDING_URLS_HELP", "https://github.com/nautobot/nautobot/wiki"),
+}
```

Comment:
When I see this, I automatically think: _Oh, could we make this section so that one could place all kinds of named links here if desired_? I‘m not sure this is desirable, but I did want to mention it as something that some might desire.